validate.js

```diff
@@ -0,0 +1,25 @@
+const showMessage = (message) => {
+  const msg = document.getElementById('errorMsg');
+  msg.innerText = message;
+};
+const validateEmail = (input) => {
+  const emailRegex = /^[a-z0-9]+@[^\s]+\.[^\s]+$/g;
+  const email = input.value.trim();
+  if (emailRegex.test(email)) {
+    return true;
+  }
+  return false;
+};
```

Comment:
Currently, it shows an error message if the user enters an email address with some capital letter before the period(.) in the address, and if the user types an email address with some capital letter after the period it does not show any error message else it submits the form. Consider reviewing it as it's a project requirement.

## MORE CLARIFICATION:-

Shows an error message on this email 👋 

`MrAbdulali2003@gmail.com`

But not for this email address with capital letters after the period 👎 

`mrabdulali2003@gmail.coM`